packages/react-is/src/ReactIs.js

```diff
@@ -125,6 +127,9 @@ export function isFragment(object: any) {
 export function isLazy(object: any) {
   return typeOf(object) === REACT_LAZY_TYPE;
 }
+export function isDebugTraceMode(object: any) {
```

Comment:
Let's drop this. Don't want another `isAsyncMode`. :D

Reply:
Sure. Although it feels weird leaving `react-is` in an incomplete/broken state.

Reply:
This would also break DevTools display but maybe that's.. okay